setup.cfg

```diff
@@ -40,6 +40,7 @@ include_package_data = True
 install_requires =
     antlr4-python3-runtime ==4.7.2
     cftime >=1.2
+    jinja2
```

Comment:
This is a top level dependency, so let's be explicit rather than relying on `antlr4` pulling this in.